### PR TITLE
Add release notes for edge-19.1.2 release

### PR DESCRIPTION
**Pull request by siggy**
Depends on #2037, #2038, #2063, #2066, ~#2086~, #2087, #2089, #2105

Signed-off-by: Andrew Seigner <siggy@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## edge-19.1.2
+
+* Controller
+  * Retry support! Introduce an `isRetryable` property to service profiles to
+    enable configuring retries on a per-route basis
+  * Add Daemonset support to Web UI, Grafana, and cli `stat` command (thanks,
+    @zknill!)
+* Web UI
+  * Add `linkerd uninject` command to remove the Linkerd proxy from a Kubernetes
```
Comment (alpeb) [inline CHANGES.md]: `linker uninject` should go under the CLI section :wink:

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+    @zknill!)
+* Web UI
+  * Add "meshed" and "no traffic" badges on the resource detail pages
+  * Fixed `linkerd dashboard` to maintain proxy connection when browser open
+    fails
+  * Fixed JavaScript bundling to avoid serving old versions after upgrade
+* CLI
+  * Add `linkerd uninject` command to remove the Linkerd proxy from a Kubernetes
+    config
+  * Add `linkerd logs` command to surface logs from any container in the Linkerd
```
Comment (wmorgan) [inline CHANGES.md]: Shoudl we give a shoutout to stern?

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+* Controller
+  * Retry support! Introduce an `isRetryable` property to service profiles to
+    enable configuring retries on a per-route basis
+  * Add Daemonset support to Web UI, Grafana, and cli `stat` command (thanks,
+    @zknill!)
+* Web UI
+  * Add "meshed" and "no traffic" badges on the resource detail pages
+  * Fixed `linkerd dashboard` to maintain proxy connection when browser open
+    fails
+  * Fixed JavaScript bundling to avoid serving old versions after upgrade
```
Comment (klingerf) [inline CHANGES.md]: This is super nit picky, different bullet points have different tenses. "Introduce" and "Add" are present tense, but "Fixed" and "Improved" are past. I'd prefer to use the same tense throughout, but totally your call for which one you prefer.

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+* CLI
+  * Add `linkerd uninject` command to remove the Linkerd proxy from a Kubernetes
+    config
```
Comment (klingerf) [inline CHANGES.md]: As a separate bullet point, we should also mention that `linkerd inject` now support re-injecting resources that have already been injected.

**Pull request merged**